packages/widget-v2/src/constants/graz.ts

```diff
@@ -0,0 +1,22 @@
+import { chains as _mainnetChains, assets as _mainnetAssets } from "chain-registry/mainnet"
+import { chains as _testnetChains, assets as _testnetAssets } from "chain-registry/testnet"
+import { chainRegistryChainToKeplr } from "@chain-registry/keplr";
+import { ChainInfo } from "@keplr-wallet/types";
+
+
+
+export const mainnetChains = _mainnetChains.map((chain) => {
+  try {
+    return chainRegistryChainToKeplr(chain, _mainnetAssets)
+  } catch (_error) {
+    return false
+  }
```

Comment:
this is due a error thrown by `chainRegistryChainToKeplr` function, so we need a try catch